packages/@aws-cdk-testing/framework-integ/test/aws-codepipeline-actions/test/integ.pipeline-elastic-beanstalk-deploy.ts

```diff
@@ -83,7 +83,7 @@ const beanstalkEnv = new elasticbeanstalk.CfnEnvironment(stack, 'beanstlk-env',
   applicationName: beanstalkApp.applicationName!,
   environmentName: 'codepipeline-test-env',
   // see https://docs.aws.amazon.com/elasticbeanstalk/latest/platforms/platforms-supported.html#platforms-supported.nodejs
-  solutionStackName: '64bit Amazon Linux 2023 v6.2.1 running Node.js 20',
+  solutionStackName: '64bit Amazon Linux 2023 v6.2.2 running Node.js 20',
```

Comment:
@Leo10Gama .. did you manually do this change?

Reply:
Yes, I manually changed it. According to their [platform history](https://docs.aws.amazon.com/elasticbeanstalk/latest/platforms/platform-history-nodejs.html#platform-history-2024-10-04), this value is not current, so no longer supported. Trying to deploy with it gave me this error:
```
❌  aws-cdk-codepipeline-elastic-beanstalk-deploy failed: 
The stack named aws-cdk-codepipeline-elastic-beanstalk-deploy failed creation, it may need to be manually deleted from the AWS console: 
ROLLBACK_COMPLETE: 
Resource handler returned message: 
"No Solution Stack named '64bit Amazon Linux 2023 v6.2.1 running Node.js 20' found. (Service: ElasticBeanstalk, Status Code: 400, Request ID: <ID>)" (RequestToken: <TOKEN>, HandlerErrorCode: InvalidRequest)
  FAILED     aws-codepipeline-actions/test/integ.pipeline-elastic-beanstalk-deploy-codepipeline-elastic-beanstalk-deploy/DefaultTest (undefined/us-east-1) 211.99s
      Integration test failed: Error: Command exited with status 1
```
Updating the value let the test pass.